site/src/pages/tutorials/tutorial-blog-rest-create.mdx

```diff
@@ -0,0 +1,192 @@
+---
+menuTitle: "Implement CREATE"
+order: 4
+type: step
+tags:
+  - rest
+  - request
+  - request-converter
+level: basic
+---
+
+# Implementing CREATE operation
+
+In this step, you'll write a service method for creating a blog post. By completing this step, you'll learn to map your service with the HTTP POST (<type://@Post>) method and make your own request converter (<type://@RequestConverter>).
+
+<TutorialSteps current={4} />
+
+## What you need
+
+You must have the following files ready for creating a blog post. You can always [download](https://github.com/line/armeria-examples/blob/master/tutorials/rest-api-annotated-service/src/main/java/example/armeria/server/blog/) the full version, instead of creating one yourself.
+
+- Main.java from [Step 1. Create a server](/tutorials/tutorial-blog-rest-create-server)
+- BlogPost.java from [Step 2. Prepare a data object](/tutorials/tutorial-blog-rest-prepare-data-object)
+- BlogService.java from [Step 3. Add services to a server](/tutorials/tutorial-blog-rest-add-services-to-server)
+
+## 1. Map HTTP method
+
+Let's start [mapping the HTTP POST method](/docs/server-annotated-service#mapping-http-service-methods) with our service method:
+
+1. Declare a service method, `createBlogPost()`, in the class `BlogService`.
+2. Map this service method with the HTTP POST method by adding the <type://@Post> annotation.
+3. Bind the endpoint `/blogs` to the method.
+
+```java filename=BlogService.java highlight=6
+import com.linecorp.armeria.server.annotation.*;
+
+public final class BlogService {
+  ...
+
+  @Post("/blogs")
+  public void createBlogPost(BlogPost blogPost) {}
+}
+```
+
+## 2. Handle parameters
+
+Let's receive blog post information through a request body. Armeria's [request converter]
+(/docs/server-annotated-service#converting-an-http-request-to-a-java-object) converts request parameters in HTTP messages into Java objects for you. In the request converter, we define what keys of a JSON object to map with what properties of a Java object.
```

Comment:
An href link for the following link will not be generated cause there is a new line between `[..]` and `(...)`.
```suggestion
Let's receive blog post information through a request body. Armeria's [request converter](/docs/server-annotated-service#converting-an-http-request-to-a-java-object)
converts request parameters in HTTP messages into Java objects for you. In the request converter, we define what keys of a JSON object to map with what properties of a Java object.
```